lisa/sut_orchestrator/azure/features.py

```diff
@@ -182,6 +182,20 @@ def _execute(self, wait: bool, operator: str, **kwargs: Any) -> None:
         if wait:
             wait_operation(operation, failure_identity="Start/Stop")
 
+    def get_status(self) -> str:
+        platform: AzurePlatform = self._platform  # type: ignore
+        compute_client = get_compute_client(platform, api_version="2021-07-01")
```

Comment:
try if latest version is ok, then use the latest version, not hard code and copy the previous versions.